### PR TITLE
refactor: revert API breaking change

### PR DESCRIPTION
**Pull request by a-cordier**

### CODE REVIEW

**Review by Okhelifi — COMMENTED** [review]
Look good for me. 👍🏼

**Review by Okhelifi — APPROVED** [review]

File: config/samples/context/debug/api-context-with-credentials.yml [hunk 1]
```diff
@@ -14,18 +14,15 @@
 
 # Use this context if you are running APIM and GKO locally
 apiVersion: gravitee.io/v1alpha1
-kind: ApiContext
+kind: ManagementContext
```
[the remark this answers is not in the document]
Comment (a-cordier) [reply config/samples/context/debug/api-context-with-credentials.yml:17]: 👍

File: config/manager/kustomization.yaml [hunk 2]
```diff
@@ -26,5 +26,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: controller
+  newName: k3d-graviteeio.docker.localhost:12345/gko
```
[the remark this answers is not in the document]
Comment (a-cordier) [reply config/manager/kustomization.yaml:29]: 👍

**Pull request merged**